pointers/pointers.c

```diff
@@ -9,6 +9,11 @@
 void string_copy(char *x, char *y)
 {
 
+    while(*x != *y) {
+        *x = *y;
+        y++;
```

Comment:
This is just going to write the first memory address of x to y and not stop. *Hint* when do strings terminat in C? what character do you have to hit to know you've hit the end of the string?